include/cinder/GeomIo.h

```diff
@@ -528,9 +528,11 @@ class CI_API Torus : public Source {
 	//! Allows you to twist the torus along the ring. The \a offset is in radians.
 	Torus&	twist( unsigned twist, float offset ) { mTwist = twist; mTwistOffset = offset; return *this; }
 	//! Specifies the major and minor radius as a ratio (minor : major). Resulting torus will fit unit cube.
-	Torus&	ratio( float ratio ) { ratio = math<float>::clamp( ratio ); mRadiusMajor = 1; mRadiusMinor = 1 - ratio; return *this; }
-	//! Specifies the major and minor radius separately.
+	Torus&	ratio( float ratio ) { ratio = math<float>::clamp( ratio ); mRadiusMajor = 1; mRadiusMinor = ratio; return *this; }
```

Comment:
The docs are a nice improvement IMO. though isn't it a breaking change? Though I'm not against it in this case, I don't think the `ratio()` method is used as much. @paulhoux what do you think?

Reply:
It is a breaking change indeed, but I think `ratio()` was not working according to its documentation. if `ratio = minor / major` and `major = 1` then `minor` should be equal to `ratio` and not `1 - ratio`.